### ISSUE TITLE
Peer dependency error

### ISSUE THREAD

**Issue by hongsa** [DataDog/dd-sdk-reactnative#84]
I can't install library

Below Error Code

```
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR!   peer react@"*" from @react-navigation/native@5.9.4
npm ERR!   node_modules/@react-navigation/native
npm ERR!     @react-navigation/native@"^5.9.4" from the root project
npm ERR!     peer @react-navigation/native@"^5.6.0" from dd-sdk-reactnative@1.0.0-beta3
npm ERR!     node_modules/dd-sdk-reactnative
npm ERR!       dd-sdk-reactnative@"*" from the root project
npm ERR!   1 more (react-native)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from dd-sdk-reactnative@1.0.0-beta3
npm ERR! node_modules/dd-sdk-reactnative
npm ERR!   dd-sdk-reactnative@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/hongsa/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hongsa/.npm/_logs/2021-07-22T02_38_52_419Z-debug.log
```

My react-native version is 0.64.1

**Comment by 0xnm** [comment]
Hello @hongsa. This is npm-specific behavior related to the peer dependencies handling. You can solve the issue by using `--force` or `--legacy-peer-deps` flags as suggested in the meantime.

**Comment by 0xnm** [comment]
Closed by #86
